History.rdoc

```diff
@@ -1,3 +1,7 @@
+=== 0.5.1 / 2017-06-23
+
+* Constrain rgeo to compatible versions (< 0.5)
```

Comment:
Good catch. 0.6.0 is current rgeo so there is no longer a constraint on rgeo at all and nothing to document.

However, I realized that this PR exposes the mysql constraint to users. See the updated title and description.